### PR TITLE
[16.0][IMP] module_change_auto_install: also allow using environment variables

### PR DESCRIPTION
**Pull request by vincent-hatakeyama**

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
makes sense !

**Review by houzefa-abba — APPROVED** [review]

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Could you add an exemple of how to use environment variable in the configure section, specially for the dict syntax ?

**Comment by vincent-hatakeyama** [comment]
I added information on how to use the environment variables too.

**Comment by legalsylvain** [comment]
Thanks !

/ocabot merge minor

**Comment by legalsylvain** [comment]
@vincent-hatakeyama could you take a look ?

**Comment by vincent-hatakeyama** [comment]
> @vincent-hatakeyama could you take a look ?

I’ve remove the rst that seems to cause trouble.

**Review by legalsylvain — APPROVED** [review]
/ocabot merge minor

**Comment by vincent-hatakeyama** [comment]
The tests in error are unrelated, they are from module attachment_synchronize.

**Comment by legalsylvain** [comment]
@vincent-hatakeyama : waiting for https://github.com/OCA/storage/pull/372

**Comment by SirAionTech** [comment]
> @vincent-hatakeyama : waiting for [OCA/storage#372](https://github.com/OCA/storage/pull/372)

@legalsylvain it is merged

**Comment by legalsylvain** [comment]
Thanks ! 

/ocabot merge patch

**Review by SirAionTech — APPROVED** [review]
Thanks for the PR!
Only code review

**Comment by legalsylvain** [comment]
@vincent-hatakeyama could you rebase ?

File: module_change_auto_install/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -4,15 +4,21 @@
...
+  The environment variable ``ODOO_MODULES_AUTO_INSTALL_ENABLED`` can also be set.
+
+The values in the configuration file takes precedence over the environment variable
+values.
 
 **Typical Settings**
 
-.. code-block:: shell
+.. code-block:: cfg
+   :emphasize-lines: 1
```
Comment (SirAionTech) [inline module_change_auto_install/readme/CONFIGURE.rst]:
```suggestion
```
This is probably the line causing the merge fails, can it be removed?

**Review by SirAionTech — CHANGES_REQUESTED** [review]

**Comment by vincent-hatakeyama** [comment]
> @vincent-hatakeyama could you rebase ?

I’ve rebased after removing the line emphasis.

**Review by SirAionTech — APPROVED** [review]

**Comment by legalsylvain** [comment]
Merge, fourth try...

/ocabot rebase

**Pull request merged**